src/main/kotlin/wu/seal/jsontokotlin/MakeKotlinClassAction.kt

```diff
@@ -144,6 +145,23 @@ class MakeKotlinClassAction : AnAction("MakeKotlinClass") {
                 if (offset == 0) {
                     offset = document.textLength
                 }
+                val lastPackageKeywordLineEndIndex = try {
+                    "^[\\s]*package.+\n$".toRegex(RegexOption.MULTILINE).findAll(document.text).last().range.endInclusive
```

Comment:
Do we need a space after `package` in the regex in case that other statement starts with `package`? i.e. `"^\\s*package\\s+.+\n$"`